src/modules/decisions/components/form/FormContainer.tsx

```diff
@@ -119,42 +120,34 @@ class FormContainer extends React.Component<FormContainerProps, FormContainerSta
         queryCategories: formattedCategories
       });
     }
+    const initialDms = getQueryParam(SearchComponents.DM);
 
-    const initialDm = getQueryParam(SearchComponents.DM);
-    if(initialDm) {
+    if(initialDms) {
       const { t } = this.props;
-      let dm = JSON.parse(initialDm);
-
-      let foundDm = SectorMap.find((element) => {
-        if (element.label === dm) {
-          return true;
+      let dmsString = JSON.parse(initialDms);
+      let dms = dmsString.split(',');
+      // TODO
```

Comment:
Is there something unfinished here? If not, we should just remove the TODO comment.